bfd/src/test/java/gov/cms/ab2d/bfd/client/ConvertXmlToJson.java

```diff
@@ -0,0 +1,45 @@
+package gov.cms.ab2d.bfd.client;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+import ca.uhn.fhir.rest.api.EncodingEnum;
+import org.apache.commons.io.FileUtils;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.MissingResourceException;
+
+public class ConvertXmlToJson {
```

Comment:
Maybe some documentation on when to run this and what it is for? I understand based on the context that this is how you convert sample data to JSON but...

Reply:
It's how I converted the tests so they would work with JSON instead of XML. It's not needed for a test but I thought it could be useful in the future.